theano/gpuarray/tests/test_dnn.py

```diff
@@ -2158,11 +2158,11 @@ def dconvi(kern, topgrad, imshp):
 
 class Cudnn_grouped_conv(Grouped_conv_noOptim):
     mode = mode_with_gpu
-    conv2d = staticmethod(dconv2d)
-    conv2d_gradw = staticmethod(dconv2dw)
-    conv2d_gradi = staticmethod(dconv2di)
-    conv2d_op = staticmethod(dnn.GpuDnnConv)
-    conv2d_gradw_op = staticmethod(dnn.GpuDnnConvGradW)
-    conv2d_gradi_op = staticmethod(dnn.GpuDnnConvGradI)
+    conv2d = dconv2d
+    conv2d_gradw = dconv2dw
+    conv2d_gradi = dconv2di
```

Comment:
These functions probably need the staticmethod otherwise, it will attempt to convert them to class methods.  But the classes don't need them.